pkg/genericapiserver/genericapiserver_test.go

```diff
@@ -89,14 +85,6 @@ func TestNew(t *testing.T) {
 	serviceReadWriteIP, _ := ipallocator.GetIndexedIP(serviceClusterIPRange, 1)
 	assert.Equal(s.ServiceReadWriteIP, serviceReadWriteIP)
 	assert.Equal(s.ExternalAddress, net.JoinHostPort(config.PublicAddress.String(), "6443"))
-
-	// These functions should point to the same memory location
```

Comment:
> Was there any value in these tests? They are gone now.

No.  It was just wiring that was unused and no longer exists since we only care about the transport.